### PR TITLE
Fix - 'Settings' object missing attribute (#41)

### PR DESCRIPTION
**Pull request by guam68**
Hey there. The added code will set both "CHOICES_SEPARATOR" and "ROOT" when the package is initialized if they are not defined in the users settings.

My only concern is that the user's ROOT will be set to the same directory as the survey app. I've been looking through the code to see if this will cause any issues but couldn't find anything obvious. Please let me know if this will be an issue or if there are any other concerns.

### CODE REVIEW

File: survey/__init__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def set_default_settings():
+    try:
+        from django.conf import settings
+        from . import settings as app_settings
+
+        for setting in dir(app_settings):
+            if setting == "CHOICES_SEPARATOR" or setting == "ROOT":
```
Comment (Pierre-Sassoulas) [inline survey/__init__.py]:
```suggestion
            if setting in ["CHOICES_SEPARATOR", "ROOT"]:
```

**Comment by Pierre-Sassoulas** [comment]
After checking the commit history in detail it seems there is 8 commit because of a problem with the rebase and there should be only one commit. I advise you to do the following commands:

```
git remote add psass https://github.com/Pierre-Sassoulas/django-survey.git
git rebase psass/master
# git rebase --skip in case of conflict
````

Or maybe :
```
git remote add psass https://github.com/Pierre-Sassoulas/django-survey.git
git checkout psass/master -b dev
git checkout dev
git cherry-pick 90243e57545724e4fd29f86866d24257f14666c0
# And then check if the result is sane before replacing master with dev
````

**Comment by guam68** [comment]
Sounds good. I will try to start work on some tests this weekend. 

I apologize about the conflicts. I believe this was a result of me working straight off of my master branch. I'll make sure to avoid this in the future. Thanks

**Comment by Pierre-Sassoulas** [comment]
No problem, Thank you for rebasing. I think you made two merge requests one after the other, I accepted the first one and you merged your own merged merge request back into your branch. :) 

I need to check this one, more thoroughly, I'll probably merge this week-end or the next one.

**Comment by guam68** [comment]
Hey I decided to separate this code into two different pulls for each issue. The CHOICES_SEPARATOR fix works fine but the setting of ROOT is causing issues elsewhere when running through the tests. When the ROOT gets reset to the survey directory, the test for updating the csv files fails due to ROOT no longer pointing to the correct location of the csv folder. I'll have to figure something else out to get that working.

**Comment by Pierre-Sassoulas** [comment]
Good call. Now that I think of it `ROOT` is an internal value to avoid changing the path everytime we move a file around. Users should not be able to change it.

**Comment by Pierre-Sassoulas** [comment]
Sorry for the long time I took in order to check the MR. and thank you for your work, this was really needed for a lot of bugs we have right now. (#41 , #73 , #27 ...) so much appreciated.

**Pull request merged**